admin/controllers/service-manager.js

```diff
@@ -8,10 +8,8 @@ const settingsValidator = require('../lib/validator/settings-validator')
 const checkWindowValidator = require('../lib/validator/check-window-validator')
 const checkWindowErrorMessages = require('../lib/errors/check-window')
 const checkWindowService = require('../services/check-window.service')
-const dateService = require('../services/date.service')
 const checkWindowDataService = require('../services/data-access/check-window.data.service')
```

Comment:
my editor tells me this is no longer used.

Reply:
@GuyHarwood yea spotted that as well. I accidentally approved it when I was resolving conflicts, will be deleted on my next commit.